package.json

```diff
@@ -99,6 +99,7 @@
     "spdx-satisfies": "^4.0.0",
     "tar": "^4.4.4",
     "through2": "^2.0.3",
+    "ts-api-guardian": "^0.3.0",
```

Comment:
I don't think you want to do it this way. Our repo is Bazel and so is the one owning that library. So just add a WORKSPACE dependency and call the `ts_api_guardian_test` https://github.com/angular/angular/blob/master/tools/ts-api-guardian/index.bzl#L22

Reply:
Updated, though I still have an error which I am trying to see whats the cause of it 

```
no such package 'tools/ts-api-guardian': BUILD file not found on package path and referenced by '//etc/api:angular_devkit_architect_api.accept'
```

Reply:
Update, though I still have a bazel error, will try to see what’s the root cause later.